### PR TITLE
Cleared all comments

### PR DESCRIPTION
**Pull request by anilbabukb**

### CODE REVIEW

File: VS_Git_Rebase/SubmittalModel.cs [hunk 1]
```diff
@@ -2,16 +2,6 @@
...
-        //Lastest change
-        //Current change
-        //Yet Another change
-        //New Commit
-        //changes after submitting the pull request
-        //Check whether origin head gets deleted while fetch
-        //For pull request
-        //For another pull request
-        // Yet another pull request
+        //Cleared all comments
```
Comment (noufionline) [inline VS_Git_Rebase/SubmittalModel.cs]: Why are you leaving so much space after comment?
Remove empty space

File: VS_Git_Rebase/SubmittalModel.cs [hunk 2]
```diff
@@ -2,16 +2,6 @@
 {
     class SubmittalModel
     {
```
Comment (noufionline) [inline VS_Git_Rebase/SubmittalModel.cs]: Test Comment

File: VS_Git_Rebase/SubmittalModel.cs (hunk 1 above)
Comment (anilbabukb) [reply VS_Git_Rebase/SubmittalModel.cs]: You missed the point there is no space.. it is showing the diff

Comment (noufionline) [reply VS_Git_Rebase/SubmittalModel.cs]: Ok Got It.

File: VS_Git_Rebase/SubmittalModel.cs [hunk 3]
```diff
@@ -2,16 +2,6 @@
 {
     class SubmittalModel
```
Comment (noufionline) [inline VS_Git_Rebase/SubmittalModel.cs]: Please consider adding public before class

**Pull request merged**